### DISCUSSION TITLE
ERG Mode for iConsole Elliptical

### DISCUSSION THREAD

**Discussion by qordaz** [cagnulein/qdomyos-zwift#523] [Q&A]
Hello, as I wrote in the fb group today, I would love to see ERG mode working on the Elliptical in Zwift.
I am using an iConsole device, and for this app to work in Zwift I am enabling the Toorx/iConsole Bike option.

I would love to be able to have Zwift control the resistance on my Elliptical when I start climbing, and also shift down again when going downhill. It feels a little weird as it is.

Right now it seems as if Zwift adjusts my speed down when I go uphill, and not setting the resistance up. Even with ERG mode enabled in Bike Trainer options.

Thank you again for a wonderful app, hoping it will be even more wonderful :)

**Comment by cagnulein** [comment]
hi @qordaz first of all, could you share a debug log from your current setup?
Thanks

**Reply by qordaz** [reply]
Sure, not sure how though. Is this a debug log I need to export from the android QZ app, or is it from Zwift?

**Reply by cagnulein** [reply]
from QZ https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F
